dockerfiles/run/guice/cassandra-ldap/destination/admin.json

```diff
@@ -0,0 +1,5 @@
+{
+  "web": {
+    "bindAddress": "0.0.0.0"
```

Comment:
localhost would be sufficient, no?

Reply:
You can not access it out of their docker container without that

Reply:
right, this must be configured when launching the image